portfolio/src/main/webapp/index.html

```diff
@@ -158,6 +158,13 @@ <h2 class="title mb-3 font-weight-bold">
 
   <script>
 
+    function mobileAlert() {
+            alert("Hello Dear Stranger!\nAlex here, I noticed that you are checking my portfolio from your mobile and I wanted to suggest you to open it from your computer instead\nCheers!");
+        }
+
+    if (/Mobi/.test(navigator.userAgent)) {
+      mobileAlert();
+    }
```

Comment:
Similar to my comments in your first PR, I think these should be in a separate script file. Once you make the changes in first PR, you can add this part to that same file. If it is hard to organize the code this way, you can skip my comments but keep it in mind for future practices.